media-libs/opencolorio/files/opencolorio-2.1.2-musl-stol.patch

```diff
@@ -0,0 +1,19 @@
+# Fix strtol_l missing on musl. On musl libc use strtol
+# Taken from Alpine Linux, please refer:
+# https://git.alpinelinux.org/aports/tree/community/opencolorio/0002-fix-strtol.patch
```

Comment:
The https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/1624 issue also seems to be related.